src/app.js

```diff
@@ -156,15 +159,39 @@ var run = function () {
   chromeCloudSync()
 
   // ----------------- editor ----------------------
-
   var editor = new Editor()
-
-  // ----------------- tabbed menu -------------------
+  // -------------- fileExplorer ------------------
+  // @NOTE HTML#ID creates window['<ID name>'] element as a global
+  // var api = fileExplorer(window.treeview, files, editor)
+  fileExplorer(window.treeview, files, editor)
```

Comment:
* I'm generally avoiding `new` and work only with factory functions.
* I'm not sure what `var fileExplorer` should contain - currently `fileExplorer(..)` returns an empty api object.
* The fileExplorer currently needs to know it's container so it can properly update itself, so i would still pass the same three arguments to fileExplorer